src/Microcks.Testcontainers/MicrocksBuilder.cs

```diff
@@ -0,0 +1,208 @@
+namespace Microcks.Testcontainers;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+public sealed class MicrocksBuilder : ContainerBuilder<MicrocksBuilder, MicrocksContainer, MicrocksConfiguration>
+{
+	/// <summary>
+	/// Image name for the Microcks container.
+	/// </summary>
+	public const string MicrocksImage = "quay.io/microcks/microcks-uber";
+
+	/// <summary>
+	/// HTTP port for the Microcks container.
+	/// </summary>
+	public const ushort MicrocksHttpPort = 8080;
+
+	/// <summary>
+	/// GRPC port for the Microcks container.
+	/// </summary>
+	public const ushort MicrocksGrpcPort = 9090;
+
+	private List<string> _snapshots;
+
+	private List<string> _mainRemoteArtifacts;
+	private List<string> _mainArtifacts;
+	private List<string> _secondaryArtifacts;
+
+	private List<Model.Secret> _secrets;
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="MicrocksBuilder" /> class.
+	/// </summary>
+	public MicrocksBuilder()
+		: this(new MicrocksConfiguration())
+	{
+		DockerResourceConfiguration = Init().DockerResourceConfiguration;
+	}
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="MicrocksBuilder" /> class.
+	/// </summary>
+	/// <param name="resourceConfiguration">The Docker resource configuration.</param>
+	private MicrocksBuilder(MicrocksConfiguration resourceConfiguration)
+		: base(resourceConfiguration)
+	{
+		DockerResourceConfiguration = resourceConfiguration;
+	}
+
+	/// <inheritdoc />
+	protected override MicrocksConfiguration DockerResourceConfiguration { get; }
+
+	/// <inheritdoc />
+	public override MicrocksContainer Build()
+	{
+		Validate();
+		var container = new MicrocksContainer(DockerResourceConfiguration);
+
+		container.Started += (_, _) => ContainerStarted(container);
```

Comment:
Looks like cryptic notation to me 😂

Reply:
Oh! 
It's a subscribe to event with anonymous function.

It's similar to: 
```csharp
container.Started += MyAnonymousFunc;

public void MyAnonymousFunc(object sender, EventArgs args) 
{
    ContainerStarted(container)
}
```

I don't use sender and args, "_" else i write: 
```csharp
container.Started += (sender, eventArgs) => ContainerStarted(container);
```

Reply:
Thanks for the explanation!